doc/cephadm/install.rst

```diff
@@ -130,9 +130,9 @@ command.  There are several ways to do this:
 
     # cephadm shell
 
-* It may be helpful to create an alias::
+* To execute ``ceph`` commands, you can also runn commands like so::
```

Comment:
typo:
```suggestion
* To execute ``ceph`` commands, you can also run commands like so::
```

It can also be:
```suggestion
* ``ceph`` commands can also be run as::
```